base/shared/src/main/scala/scalaz/data/Kleisli.scala

```diff
@@ -0,0 +1,35 @@
+package scalaz
+package data
+
+sealed trait KleisliModule {
+  type Kleisli[F[_], A, B]
+
+  def run[F[_], A, B](k: Kleisli[F, A, B]): A => F[B]
+
+  def wrap[F[_], A, B](k: A => F[B]): Kleisli[F, A, B]
+
+  def hoist[F[_], G[_], A, B](k: Kleisli[F, A, B])(η: F ~> G): Kleisli[G, A, B]
+
+  def compose[F[_], A, B, C](
+    k: Kleisli[F, A, B],
+    j: Kleisli[F, B, C]
+  )(implicit M: Monad[F]): Kleisli[F, A, C]
```

Comment:
By convention, `compose` has the opposite parameter order. This one would be called `andThen` or `>>>`.

Reply:
Agreed